.github/workflows/build.yml

```diff
@@ -76,6 +114,86 @@ jobs:
     steps:
     - uses: actions/checkout@v2
 
+    - name: Use Node.js 12.x
+      uses: actions/setup-node@v1
+      with:
+        node-version: '12.x'
+
+    - name: Cache Node.js modules
+      id: node-cache
+      uses: actions/cache@v2
+      with:
+        path: node_modules
+        key: ${{ runner.OS }}-node-modules-${{ hashFiles('package-lock.json') }}
+        restore-keys: |
+          ${{ runner.OS }}-node-modules-
+          ${{ runner.OS }}-
+
+    - run: npm ci
+      name: Install dependencies
+      if: steps.node-cache.outputs.cache-hit != 'true'
+
+    - run: npm run lint:ios
+      name: Lint
+
+    - run: npm run build:ios
+      name: Build
+
+    - name: Cache Native Modules
```

Comment:
This should no longer be necessary if we're only building the iOS source in this job and should be moved to the `package` job to run before the `npm run package` step